ui/imports/utils/Utils.qml

```diff
@@ -895,11 +895,7 @@ QtObject {
         return getCommunityChannelShareLink(communityId, channelId)
     }
 
-    function getCommunityDataFromSharedLink(link) {
-        const index = link.lastIndexOf("/c/")
-        if (index === -1)
-            return null
-
```

Comment:
I removed the validation here because it just caused us to double the amount of validation and was error bound, especially if new types of links are added, we need to add a new line or update a regex.

There is already validation done in `ImportCommmunityPopup` (the only place this function is called) and we also have a try/catch below that makes sure that if something goes wrong in the backend (weird unsupported link), we still return a correct value.